consume.go

```diff
@@ -247,9 +258,10 @@ func (cmd *consumeCmd) parseFlags(as []string) consumeArgs {
 	flags.DurationVar(&args.timeout, "timeout", time.Duration(0), "Timeout after not reading messages (default 0 to disable).")
 	flags.BoolVar(&args.verbose, "verbose", false, "More verbose logging to stderr.")
 	flags.StringVar(&args.version, "version", "", "Kafka protocol version")
-	flags.StringVar(&args.encodeValue, "encodevalue", "string", "Present message value as (string|hex|base64), defaults to string.")
+	flags.StringVar(&args.encodeValue, "encodevalue", "string", "Present message value as (string|hex|base64|binary|proto), defaults to string.")
 	flags.StringVar(&args.encodeKey, "encodekey", "string", "Present message key as (string|hex|base64), defaults to string.")
-
+	flags.StringVar(&args.pType, "ptype", "", "Protobuff type name to be used for decoding encodevalue=proto")
+	flags.StringVar(&args.rpcProcessCmd, "rpcprocesscmd", "", "command to run to start RPC process for protobuff deserialization")
```

Comment:
not sure it should be the responsibility of the kt binary to *start* the RPC process, although i'm fine with the idea of *having* an RPC process